tests/test_experiment.py

```diff
@@ -1565,7 +1565,7 @@ def test_run_experiment_lr_compare():
     yield check_report, html_report
 
 
-def test_run_experiment_lr_compare():
+def test_run_experiment_lr_compare_with_h2():
```

Comment:
thank you @aloukina for catchingthis. I thought I had made the name unique, I somehow missed it.